### ISSUE TITLE
problem when calculating psd using pburg method with "real" signal

### ISSUE THREAD

**Issue by rhenanbartels** [cokelaer/spectrum#24]
Hi,  
I was trying to estimate the psd of a sinusoidal signal and I got the following exception:
```
    *** TypeError: slice indices must be integers or None or have an __index__ method
```
I was using Python 3.6.0 and I've ran the following code:

```
import numpy as np
import spectrum
t = np.linspace(0, 1, 1000)
s = np.sin(2 * np.pi * 5 * t)
p = spectrum.pburg(s, order=16, NFFT=None, sampling=1000)
p()
```
The traceback:

```
    127         #self.psd = psd
    128         if self.datatype == 'real':
--> 129             newpsd  = psd[0:self.NFFT/2]*2
    130             newpsd[0] /= 2.
    131             newpsd = numpy.append(newpsd, psd[-1])

TypeError: slice indices must be integers or None or have an __index__ method
```

The same happened when I've used an integer for NFFT (i.e 256)
```
import numpy as np
import spectrum
t = np.linspace(0, 1, 1000)
s = np.sin(2 * np.pi * 5 * t)
p = spectrum.pburg(s, order=16, NFFT=256, sampling=1000)
p()
```

Thanks in advance

**Comment by olbakalis** [comment]
Try
> psd[0:int(self.NFFT/2)]*2
Python 3. Wants explicitly integers in the slices or declarations of arrays.
Thank you,
Lilly

**Comment by cokelaer** [comment]
This was fixed in an earlier release but this raise the issue which was fixed in other places in version 0.6.4
